### PR TITLE
Lady gaga the queen

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=gcFDfz9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,4 +74,5 @@ And in the free fall I will realize
 I'm better off when I hit the bottom
```
Comment (EladKohavi) [inline README.md:75]: ok

File: README.md [hunk 2]
```diff
@@ -74,4 +74,5 @@ And in the free fall I will realize
...
 
 Lady gaga the queen
+Lady gaga the queen
```
Comment (EladKohavi) [inline README.md:77]: ok

File: README.md [hunk 3]
```diff
@@ -74,4 +74,5 @@ And in the free fall I will realize
...
```
Comment (EladKohavi) [inline README.md:78]: nope

**Review by EladKohavi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (EladKohavi) [inline README.md:77]: another one

File: README.md (hunk 3 above)
Comment (EladKohavi) [inline README.md:78]: no

File: README.md (hunk 2 above)
Comment (yeelali14) [reply README.md:77]: test

File: README.md [hunk 4]
```diff
@@ -74,4 +74,5 @@ And in the free fall I will realize
 I'm better off when I hit the bottom
 
 Lady gaga the queen
```
Comment (EladKohavi) [inline README.md:76]: forgot this line

File: README.md (hunk 2 above)
Comment (EladKohavi) [inline README.md:77]: ok ok

**Review by EladKohavi — APPROVED** [review]
nice work

**Pull request closed without merging**